src/Compilers/CSharp/Test/Symbol/Symbols/ConversionTests.cs

```diff
@@ -1986,6 +1986,138 @@ static void Main()
                 Diagnostic(ErrorCode.ERR_NoImplicitConv, "0").WithArguments("int", "R"));
         }
 
+        [Fact]
+        [WorkItem(434957, "https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems?id=434957")]
+        public void SkipUserDefinedConversionsForThisArgument()
```

Comment:
Yes, these tests also (correctly) produced errors before this change. The tests made more sense with an earlier iteration of this change and are now likely redundant, and I considered removing them.